client/src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Button = (props) => (
+  <button type={props.buttonType} className={'button ' + props.className}>{props.text}</button>
+);
+
```

Comment:
You also should write 'propTypes' for every component which you writing. ( Button.propTypes = { /* proptypes */ }) )